src/index.html

```diff
@@ -10,7 +10,32 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="body">
+    <form
```

Comment:
You form currently has a default browser x in the right end. Try to find a way to remove it.

Reply:
My forms hasn't `x`.